.github/tools/gitchanges.py

```diff
@@ -49,9 +49,14 @@ def getTags():
     tags = filter(lambda x: "-" not in x, tags)
     return natsorted(tags,reverse=True)
 
-def getCommitsBetween( tagA, tagB = "HEAD" ):
-    commits = os.popen(f"git -P shortlog {tagA}..{tagB}").read().strip().splitlines()
-    return "\n".join(map( lambda x: x.replace("      ", " - "), commits ))
+def getCommitsBetween( tagA, tagB = "HEAD", dir=None):
+    if dir:
+        original_dir = os.getcwd()
+        os.chdir(dir)
```

Comment:
To be able to easily run this function in other directories I've added the `dir` parameter to temporarily change the current working directory.